package.json

```diff
@@ -59,22 +59,22 @@
     "uuid": "~3.3.2"
   },
   "devDependencies": {
-    "coveralls": "~3.0.3",
-    "istanbul": "~0.4.5",
-    "jshint": "~2.10.2",
-    "mocha": "6.1.4",
-    "nock": "10.0.6",
-    "should": "13.2.3",
-    "timekeeper": "2.2.0",
-    "watch": "~1.0.2",
-    "remark-cli": "~6.0.1",
-    "remark-preset-lint-recommended": "~3.0.3",
-    "textlint": "~11.3.1",
+    "coveralls": "~3.1.0",
+    "jshint": "~2.11.1",
+    "mocha": "8.0.1",
+    "nock": "13.0.3",
+    "nyc": "~15.1.0",
```

Comment:
Is this actually being used in scripts section?

Reply:
It is now - fixed be2edd3